data/server/callbacks.txt

```diff
@@ -16,6 +16,9 @@ forward OnSetPlayerSkinId(playerId, skinId, bool: uponNextSpawn);
 forward OnGiveWeapon(playerId, weaponId, ammo);
 forward OnResetPlayerWeapons(playerId);
 forward OnRemovePlayerWeapon(playerId, weaponId);
+forward OnPlayerCommand(playerid, cmdtext[]);
+forward OnGiveSpawnArmour(playerId);
+forward OnGiveSpawnWeapon(playerId, weaponId, multiplier);
```

Comment:
Only JavaScript functions called by Pawn have to be listed here. You can call any public Pawn function from JavaScript. (As long as it starts with On*)